lib/build.gradle.kts

```
@@ -35,6 +36,17 @@ android {
     }
 }
 
+publishing {
+    publications {
+        create<MavenPublication>("Maven") {
+            groupId = "com.xinto"
+            artifactId = "taxi"
+            version = "1.0.0"
```

```suggestion
```
Use predefined `project.version`.
You can also do this instead:
```suggestion
            version = project.version
```